privval/file.go

```diff
@@ -86,25 +86,25 @@ type FilePVLastSignState struct {
 func (lss *FilePVLastSignState) CheckHRS(height int64, round int, step int8) (bool, error) {
 
 	if lss.Height > height {
-		return false, fmt.Errorf("Height regression. Got %v, last height %v", height, lss.Height)
+		return false, fmt.Errorf("height regression. Got %v, last height %v", height, lss.Height)
```

Comment:
I'm all in favour of non-capitalized error messages (as error msg are often wrapped) but capitalized seem to be the default in this code base. I remember when I tried to change this in a PR we decided to keep them capitalized for the sake of consistency ... 
Does this still hold @melekes?

Reply:
I can revert if necessary. I was just getting to many warnings and decided to reduce some :)

Reply:
> Does this still hold @melekes?

no. let's switch to govet preferred style - non-capitalized error messages